ts/Core/Renderer/SVG/SVGElement.ts

```diff
@@ -454,9 +454,12 @@ class SVGElement implements SVGElementBase {
             y = (alignToBox.y || 0) + (alignOptions.y || 0) +
                 ((alignToBox.height || 0) - (alignOptions.height || 0)) *
                 getAlignFactor(alignOptions.verticalAlign),
-            attribs: SVGAttributes = {
-                'text-align': alignOptions?.align
-            };
+            attribs: SVGAttributes = {};
+
+        // Add text-align attribute only if option is defined, #22698
+        if (defined(alignOptions.align)) {
```

Comment:
Since `align` is of type [AlignValue](https://github.com/highcharts/highcharts/blob/a1be1023304a10a50bfc69efda5841382bc0dce8/ts/Core/Renderer/AlignObject.d.ts#L17) and can never have legit falsy values like false, 0 or `''`, a truthy check should suffice.

```suggestion
        if (alignOptions.align) {
```